### PR TITLE
chore(web-console): support writing multiple files to the same target

### PR DESCRIPTION
**Pull request by insmac**
Allow writing to the same target table name from different files.

This PR makes sure each file added to the queue is treated as a separate entity and there are no glitches in the UI.

https://github.com/questdb/ui/assets/1871646/9eef49e2-d14d-4974-a0b3-fd0d5cb9abc8

### CODE REVIEW

File: packages/web-console/src/scenes/Import/ImportCSVFiles/utils.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  }
+  return column
+}
+
+const str = () =>
+  (
+    "00000000000000000" + (Math.random() * 0xffffffffffffffff).toString(16)
+  ).slice(-16)
+
+export const uuid = () => {
```
Comment (jerrinot) [inline packages/web-console/src/scenes/Import/ImportCSVFiles/utils.ts:35]: is this how you generate a random uuid in JS? 😱

Comment (insmac) [reply packages/web-console/src/scenes/Import/ImportCSVFiles/utils.ts:35]: @jerrinot Nope, there are libraries to do so, but they all require `crypto` to be available in the system - so either HTTPS or localhost. We had users that use neither (just plain `http://`) and the system failed for them. For that reason I have just used a simple good-enough method to generate a random identifier (it's not persisted anywhere). 
I could use `new Date().getTime()` as an alternative, but you can drop multiple files to be uploaded at the same time, so there is a possibility for a trouble, I think 🙂

Comment (jerrinot) [reply packages/web-console/src/scenes/Import/ImportCSVFiles/utils.ts:35]: thanks! 
I agree it's unimportant here, I was just curious.

**Review by jerrinot — APPROVED** [review]
I tested the change locally, it does what the PR says. 
I get the idea behind the change - generate unique IDs. but I am far from a front-ent expert to say it's the best possible solution. but it looks good to me. 

I stumbled upon another weirdness while importing a CSV, but that's unrelated to this PR and it's very likely caused by a server-side bug. I'm investigating that separately.

**Pull request merged**